### PR TITLE
Add new response codes from 0.16

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Comment by gewenyu99** [comment]
Screen Shot 2022-08-19 at 5 31 29 PM
Screen Shot 2022-08-19 at 5 31 51 PM

**Review by eldadfux — COMMENTED** [review]
We need to make sure we update the appwrite/homepage, appwrite/appwrite and appwrite/docs with the new link.

File: app/views/docs/index.phtml [hunk 1]
```diff
@@ -87,7 +87,7 @@ $cols = [
                     <li><a href="/docs/pagination">Pagination</a></li>
                     <li><a href="/docs/custom-domains">Custom Domains</a></li>
                     <li><a href="/docs/permissions">Permissions</a></li>
-                    <li><a href="/docs/error-codes">Error Codes</a></li>
+                    <li><a href="/docs/response-codes">Response Codes</a></li>
```
Comment (christyjacob4) [inline app/views/docs/index.phtml:90]: reminder to update this file as well
https://github.com/appwrite/homepage/blob/cc7e8ddcbc3bf401e16e34cf8f46a2d97b65400c/app/controllers/home.php#L538-L541

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Some small comments

**Pull request merged**